app/models/appointment.rb

```diff
@@ -44,6 +44,10 @@ def created_at
     super || Time.zone.now
   end
 
+  def gmt_or_bst
```

Comment:
Doh, not paying attention - had meant to use GMT and not UTC. And I've use UTC instead od BST too.

Re: method name, yeah, I was trying to think of a noun to describe it and couldn't think of anything decent. Even `timezone` feels like it could be misconstrued, but I suppose considered in context it's accurate. Will change.